### PR TITLE
To-do-app : full website

### PR DESCRIPTION
**Pull request by Hamzaoutdoors**
# To do List app with React

- [x]  Used JavaScript events.
- [x] Use React life cycle methods.
- [x] Understand the mechanism of lifting the state up.

_thank you in advance for your feedback ❤️_

### CODE REVIEW

File: src/components/TodoContainer.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+  componentDidMount() {
+    const temp = localStorage.getItem('todos');
+    const loadedTodos = JSON.parse(temp);
+    if (loadedTodos) {
+      this.setState({
+        todos: loadedTodos,
+      });
+    }
+  }
```
Comment (YuvrajM108) [inline src/components/TodoContainer.js:25]: Nice job using lifecycle events and localStorage.

File: src/components/TodoItem.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+/* eslint-disable prefer-const */
+/* eslint-disable react/forbid-prop-types */
+
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { FaTrash } from 'react-icons/fa';
+import styles from './TodoItem.module.css';
+
+const TodoItem = (props) => {
+  const [editing, setEditing] = useState(false);
```
Comment (YuvrajM108) [inline src/components/TodoItem.js:12]: Nice use of hooks!

**Review by YuvrajM108 — APPROVED** [review]
Great job implementing the project according to the tutorial.

File: src/components/TodoContainer.js (hunk 1 above)
Comment (Hamzaoutdoors) [reply src/components/TodoContainer.js:25]: Thank you for your feedback I appreciate that 👏🏼

File: src/components/TodoItem.js (hunk 2 above)
Comment (Hamzaoutdoors) [reply src/components/TodoItem.js:12]: Thank you for your feedback I appreciate that 👏🏼

**Pull request merged**